blueocean-dashboard/pom.xml

```diff
@@ -17,7 +17,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>sse-gateway</artifactId>
-            <version>1.1</version>
+            <version>1.2-SNAPSHOT</version>
         </dependency>
     </dependencies>
 
```

Comment:
Is it ok to depend on a snapshot or is there anything that can be published first?